custom_components/adjustable_bed/beds/malouf.py

```
@@ -0,0 +1,691 @@
+        """Start listening for position notifications."""
+        # These beds don't support position feedback
+        self._notify_callback = callback
+        _LOGGER.debug("Malouf NEW_OKIN beds don't support position notifications")
+
+    async def stop_notify(self) -> None:
+        """Stop listening for position notifications."""
+        self._notify_callback = None
+
+    async def read_positions(self, motor_count: int = 2) -> None:
+        """Read current motor positions - not supported."""
+        pass
```

Ruff (0.14.13)

Unnecessary `pass` statement

Remove unnecessary `pass`

(PIE790)